### PR TITLE
Add IFLIGHT_BLITZ_F7_AIO target

### PR DESCRIPTION
**Pull request by mmosca**
This is the only fc I have on my fleet that does not have an INAV target

MCU is F745 with 1mb flash.

Baro and gyro ok.
Font upload ok.
Led strip ok. (built in led on the bottom is addressable)
M1-M4 works with DSHOOT600.
Board alignment ok.
Blackbox ok. (Needed SPI3_*_AF for it to work)
1st flight ok

Should work for a few different FC by iFlight, like the iFlight BLITZ Whoop AIO and the Defender 25 F7 AIO.

### CODE REVIEW

File: src/main/io/ledstrip.h [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 #include "config/parameter_group.h"
 #include "common/color.h"
+#include "rx/rx.h"
```
Comment (mmosca) [inline src/main/io/ledstrip.h:22]: needed for timeUs_t

**Comment by DenysHulivatenko** [comment]
> This is the only fc I have on my fleet that does not have an INAV target
> 
> MCU is F745 with 1mb flash.
> 
> Baro and gyro ok. Font upload ok. Led strip ok. (built in led on the bottom is addressable) M1-M4 works with DSHOOT600. Board alignment ok. Blackbox ok. (Needed SPI3_*_AF for it to work) 1st flight ok
> 
> Should work for a few different FC by iFlight, like the iFlight BLITZ Whoop AIO and the Defender 25 F7 AIO.

**Comment by mmosca** [comment]
This is an unofficial target, you have to build it yourself.

**Comment by DenysHulivatenko** [comment]
Unfortunately, I don't have the experience to do this myself.

Is there some general algorithm, how to do this or maybe you can send
.hex file direct?


сб, 28 жовт. 2023, 20:57 користувач Marcelo Bezerra <
***@***.***> пише:

**Pull request merged**